planning/behavior_velocity_planner/autoware_behavior_velocity_traffic_light_module/src/manager.cpp

```diff
@@ -60,7 +61,7 @@ void TrafficLightModuleManager::modifyPathVelocity(tier4_planning_msgs::msg::Pat
   stop_reason_array.header.frame_id = "map";
   stop_reason_array.header.stamp = path->header.stamp;
 
-  first_stop_path_point_index_ = static_cast<int>(path->points.size() - 1);
+  first_stop_path_point_distance_ = autoware::motion_utils::calcArcLength(path->points);
   nearest_ref_stop_path_point_index_ = static_cast<int>(path->points.size() - 1);
```

Comment:
Several `index` still exist. Is it hard to remove them?

Reply:
This PR is only for the removal of `first_stop_path_point_index_'. So this PR does not include the use of autoware_trajectory. I will replace it step-by-step.